src/App.js

```diff
@@ -1,26 +1,198 @@
+import { render } from '@testing-library/react';
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
 
-function App() {
+import visaLogo from './img/visa.png';
+import masterCardLogo from './img/master-card.svg';
+
+
+//////Iteration1//////
+class IdCardCom extends React.Component {
+  render() {
+    return(
+      <div>
+        <span>
+          <div id="left">
+          <img src={this.props.picture}></img>
+          </div>
+          <div id="right">
+            <p><strong>First name: </strong>{this.props.firstName}</p>
+            <p><strong>Last name: </strong>{this.props.lastName}</p>
+            <p><strong>Gender: </strong>{this.props.gender}</p>
+            <p><strong>Height: </strong>{this.props.height}</p>
+            {/* <p><strong>Birth</strong>{this.props.birth}</p> */}
+          </div>
+        </span>
+      </div>
+    )
+  }
+}
+
+
+//////Iteration2//////
+function LangFr(props) {
+  return(
+  <div style={{width: "800px", height:"80px", margin: "10px", border: "1px solid black" }}>
+  <h1>Bonjour François</h1>
+  </div>
+  )
+}
+
+function LangDe(props) {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div style={{width: "800px", height:"80px", margin: "10px", border: "1px solid black" }}>
+    <h1>Hallo Ludwig</h1>
     </div>
+  )
+}
+
+function Greetings(props) {
+    const language = props.lang;
+    if (language === "fr" ) {
+      return <LangFr />
+    }
+    return <LangDe /> 
+  }
+
+//////Iteration3//////
+
+
+function Random (props) {
+    const min = props.min;
+    const max = props.max;
+    const rand = Math.trunc (Math.random() * (max - min));
+
+    return(
+    <div style={{width: "800px", height:"80px", margin: "10px", border: "1px solid black" }}>
+    <h1>Random value between {props.min} and {props.max} => {rand}</h1>
+    </div>
+    )
+}
+
+//////Iteration4//////
+
+function BoxColor (props) {
+
+  function componentToHex(c) {
+    var hex = c.toString(16);
+    return hex.length === 1 ? "0" + hex : hex;
+  }
+  const hex = ("#" + componentToHex(props.r) + componentToHex(props.g) + componentToHex(props.b))
+  
+  return(
+    <div style={{backgroundColor: `${hex}`, width: "800px", height:"150px", margin: "10px", border: "1px solid black" }}>
+    <h3 style={{padding: "30px 40%"}}>rgb({props.r},{props.g},{props.b}) {hex}</h3>
+    </div>
+  )
+}
+
+//////Iteration5//////
+
+
+function CreditCard (props) {
```

Comment:
The cards looks amazing althought you should create components and organize your code.